src/Main.java

```diff
@@ -10,6 +10,7 @@ public static void main(String[] args) {
         String sinopse = "Filme de aventura relacionado a aviação";
         System.out.println(sinopse);
         System.out.println("A nota do filme é: " + notaDoFilme);
+        System.out.println("esse filme é grátis? Resposta: " + incluidoNoPlano);
```

Comment:
Não é melhor deixar o `esse` com letra maiuscula?
```suggestion
        System.out.println("Esse filme é grátis? Resposta: " + incluidoNoPlano);
```

Reply:
Acho sim